drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModel.java

```diff
@@ -991,11 +991,14 @@ protected void checkSimulation() {
      * @param instanceHeaderCellValue
      * @param columnIndex
      * @throws Exception if the given <b>instanceHeaderCellValue</b> contains a <i>dot</i> <b>OR</b> it has already been used
-     * inside the <b>group (GIVEN/EXPECT)</b> of the given column
+     * inside the <b>group (GIVEN/EXPECT)</b> of the given column <b>OR</b> instanceHeaderCellValue is <i>empty</i>
      */
-    protected void checkValidAndUniqueInstanceHeaderTitle(String instanceHeaderCellValue, int columnIndex) throws Exception {
+    protected void checkValidAndUniqueInstanceHeaderTitle(String instanceHeaderCellValue, int columnIndex) {
+        if (instanceHeaderCellValue.isEmpty()) {
```

Comment:
@yesamer 
Is there any check for `null` on `instanceHeaderCellValue` somewhere? Could we add one here - even if **currently** seems unneeded?

Reply:
@gitgabrio done!